_assets/ci/Jenkinsfile.android

```diff
@@ -47,14 +49,15 @@ pipeline {
       println("Git Branch: ${lib.gitBranch()}")
       println("Git Commit: ${lib.gitCommit()}")
       /* save and create a dir for artifacts */
-      dest = "${env.WORKSPACE}/pkg"
+      dest = "${env.WORKSPACE}/out"
       sh "mkdir -p ${dest}"
       /* for easier reuse */
       artifact = "status-go-android-${lib.suffix()}.aar"
     } } }
 
     stage('Setup') { steps { dir(env.STATUS_PATH) {
       sh 'make setup-build'
+      sh 'go get golang.org/x/tools/go/packages'
```

Comment:
A `Makefile` target would probably be neater. No harm in running this install step as part of `setup-build` target right?